Lib/pdb.py

```diff
@@ -1559,6 +1647,13 @@ def print_stack_entry(self, frame_lineno, prompt_prefix=line_prefix):
         self.message(prefix +
                      self.format_stack_entry(frame_lineno, prompt_prefix))
 
+    def print_current_inst(self, frame):
+        for inst in dis.get_instructions(frame.f_code):
```

Comment:
```suggestion
        for inst in dis.get_instructions(frame.f_code, adaptive=True):
```
See the comment on `_do_list`